src/index.html

```diff
@@ -11,22 +11,27 @@
       content="ie=edge"
     />
     <title>Document</title>
+
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body>
+    <form>
+      <input
+        class="search-input field-large"
+        type="text"
+        data-qa="keypress big"
+        placeholder="Try 'Los Angeles'"
+      />
+      <input
+        class="search-input field-small"
+        type="text"
+        data-qa="small"
```

Comment:
The `data-qa` attribute for the small input field is not correctly assigned. Please ensure it matches the task requirements.